plugins/inputs/ethtool/ethtool_test.go

```diff
@@ -305,6 +305,18 @@ func toStringMapInterface(in map[string]uint64) map[string]interface{} {
 	return m
 }
 
+func toStringMapUint(in map[string]interface{}) map[string]uint64 {
+	var m = map[string]uint64{}
+	for k, v := range in {
+		t, ok := v.(uint64)
+		if !ok {
+			errors.New("something in the interface is not a unit")
+		}
+		m[k] = t
```

Comment:
I think we have to do something with the error here. ;-) I see three possibilities

1. Make the function a member of the `ethtool` struct and use the logger to log the error message.
2. Make the function return an error.
3. Silently ignore the value.

I think 2 is not so nice at it will throw away all values. TBH I would prefer option 1, but I don't know if that's easily doable... What do you think?

Reply:
> I think we have to do something with the error here

Haha my mistake!

>  TBH I would prefer option 1, but I don't know if that's easily doable... 

I will give 1 a shot and I'll bug you if I get stuck :)